Sources/App/Views/TurboFrame.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+import Vapor
+import Plot
+
+class TurboFrame {
```

Comment:
Nit: maybe we should pick a more generic name that’s not JS library derived? Not sure if there is a good one, just feels a bit specific. Maybe just `Frame`, `PageSnippet`, `Partial`, `PartialPage` or something.

Reply:
Yea I played with names like `PageSnippet` but with the methods like `frameIdentifier` which really do need the word frame in them I figured we were already quite tightly tied to terms from the library. I guess `PageFrame` would work, but I'm not sure it's much better.

Reply:
The more I think about this, I think `TurboFrame` is the right name. This *is* very tied to that JavaScript library, so if we ever change that there will need to be a lot of changes here, and renaming the class can be one of them.